specification/batch/Azure.Batch/routes.tsp

```diff
@@ -0,0 +1,2237 @@
+  @projectedName("java", "deletePoolInternal")
+  deletePool is Ops.DeleteOperation<BatchPoolHeaders, DeleteResponseHeaders>;
+
+  @doc("Gets basic properties of a Pool.")
+  @route("/pools/{poolId}")
+  @projectedName("java", "poolExistsInternal")
+  poolExists is Ops.ResourceInfoOperationWithAdditionalResponse<
+    BatchClientRequestHeaders & BatchPoolHeaders,
+    PoolDoesNotExistResponseHeaders,
+    PoolDoesExistResponseHeaders
+  >;
+
+  @doc("Gets information about the specified Pool.")
+  @route("/pools/{poolId}")
+  @projectedName("java", "getPoolInternal")
+  getPool is Ops.ReadOperation<
+    BatchPoolHeaders & {
+      @doc("An OData $select clause.")
+      @query({
+        format: "csv",
+      })
+      @projectedName("client", "select")
+      $select?: string[];
+
+      @doc("An OData $expand clause.")
+      @query({
+        format: "csv",
+      })
+      @projectedName("client", "expand")
+      $expand?: string[];
+    },
+    BatchPool
+  >;
+
+  @summary("Updates the properties of the specified Pool.")
+  @doc("""
+This only replaces the Pool properties specified in the request. For example,
+if the Pool has a StartTask associated with it, and a request does not specify
+a StartTask element, then the Pool keeps the existing StartTask.
+""")
+  @route("/pools/{poolId}")
+  @projectedName("java", "updatePoolInternal")
+  updatePool is Ops.UpdateOperation<
+    BatchPoolHeaders & {
+      @doc("The pool properties to update.")
+      @body
+      pool: BatchPoolUpdateParameters;
+    },
+    RequestSuccesResponseHeaders
+  >;
+
+  @summary("Disables automatic scaling for a Pool.")
+  @doc("Disables automatic scaling for a Pool.")
+  @route("/pools/{poolId}/disableautoscale")
+  @projectedName("java", "disablePoolAutoScaleInternal")
+  disablePoolAutoScale is Ops.PostOperation<
+    {
+      @doc("The ID of the Pool on which to disable automatic scaling.")
+      @path
+      poolId: string;
+    },
+    {
+      @doc("A process exit code.")
+      @statusCode
+      code: "200";
+
+      @doc("The OData ID of the resource to which the request applied.")
+      @header("DataServiceId")
+      dataServiceId: string;
+    }
+  >;
+
+  @summary("Enables automatic scaling for a Pool.")
+  @doc("""
+You cannot enable automatic scaling on a Pool if a resize operation is in
+progress on the Pool. If automatic scaling of the Pool is currently disabled,
+you must specify a valid autoscale formula as part of the request. If automatic
+scaling of the Pool is already enabled, you may specify a new autoscale formula
+and/or a new evaluation interval. You cannot call this API for the same Pool
+more than once every 30 seconds.
+""")
+  @route("/pools/{poolId}/enableautoscale")
+  @projectedName("java", "enablePoolAutoScaleInternal")
+  enablePoolAutoScale is Ops.PostOperation<
+    BatchPoolHeaders & {
+      @doc("Type of content")
+      @header("content-type")
+      contentType: "application/json; odata=minimalmetadata";
```

Comment:
nit, you already have an alias for it? `MinimalMetadata`. Please avoid duplication.

Reply:
updated